### DISCUSSION TITLE
Test de prueba

### DISCUSSION THREAD

**Discussion by Burbina200** [IIC2133-PUC/2024-2#68] [Tarea 0] [answered]
Hola, buenas.

Estaba testeando mi código con los test de prueba y en algunos me pasa algo raro. Por ejemplo el Part-2-easy-1 tengo todo igual, salvo que el output de referencia tiene 3 líneas mas que mi output.

En el Part-2-medium-1 tengo todo igual hasta la línea 24.350, pero luego en el output de referencia siguen habiendo eventos (Este tiene 213.449 líneas). Me parece super raro que tantas líneas sean similares en mi output y el de referencia y que la única diferencia sea que el output de referencia proceso más eventos.

Mi pregunta es si ustedes confirmaron que están funcionando correctamente. Si es así, tendré que volver a revisar mi código.

Gracias de antemano.

**Comment by tomastrivino** [comment, marked as answer]
Hola,
entiendo que no hay problema con los tests. Trata de arreglar el part-2-easy-1 primero antes de ver el resto, ya que posiblemente te está faltando algún print al archivo.
